store/server/features/okrplanning/weeklyPriority/mutations.ts

```
@@ -23,6 +23,38 @@ const createWeeklyPriority = async (values: any) => {
+    await crudRequest({
+      url: `${OKR_AND_PLANNING_URL}/weekly-priorities/create/bulk`,
+      method: 'POST',
+      data: values,
+      headers: {
+        Authorization: `Bearer ${token}`, // Pass the token in the Authorization header
+        tenantId: tenantId, // Pass tenantId in the headers
+      },
+    });
+  } catch (error) {
+    // Handle error (optional)
+    throw error; // Re-throw error if needed for further handling
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)